### DISCUSSION TITLE
暗黑模式下修改提供的css无法实现对页面内容更改

### DISCUSSION THREAD

**Discussion by hztyx** [walinejs/waline#1195] [Q&A] [answered]
名为HTML[data-user-color-scheme="dark"]的参数
覆盖了一切参数
导致页面颜色等无法修改

**Comment by lizheming** [comment, marked as answer]
`data-user-color-scheme` 搜了下文档应该是 fluid 主题加的，和 Waline 没啥关系。你的问题是什么，看了几遍没看太懂
